ui/src/app/shared/edge/currentdata.2018.7.ts

```diff
@@ -205,6 +219,9 @@ export class CurrentDataAndSummary_2018_7 extends CurrentDataAndSummary {
             let ratio = 0;
             let maxSell = 0;
             let maxBuy = 0;
+            for (let thing of config.essType) {
+                let type = config.things[thing];
+            }

```

Comment:
What is this for? type is not used again later (and could anyway not be used later, because it is only valid within the for-block.